Packs/Yara/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,10 @@
+
+#### Indicator Types
+
+**YARA Rule**
```

Comment:
```suggestion
##### YARA Rule
```

Reply:
This is the convention for indicator types.